src/os/os_chmod_test.go

```diff
@@ -28,4 +30,26 @@ func TestChmod(t *testing.T) {
 		t.Fatalf("chmod %s %#o: %s", f.Name(), fm, err)
 	}
 	checkMode(t, f.Name(), fm)
+
+}
+
+func TestChown(t *testing.T) {
+	f := newFile("TestChown", t)
```

Comment:
What you might do first is
if runtime.GOOS == "windows" || runtime.GOOS == "plan9" {
t.Skip()
}

Then you can drop the if on line 41.